project2/style.css

```
@@ -9,3 +68,12 @@
     padding: 0;
     text-align: inherit;
 }
+
+
+.title,
+.price,
+.description,
+.rating,
+button {
+    font-family: 'Montserrat', sans-serif, bold
```

2 notes here:
1. `bold` doesn't do anything here; you need to add a `font-weight` declaration.
1. Remember to end your declarations with a semicolon.

```suggestion
    font-family: 'Montserrat', sans-serif;
    font-weight: 700;
```